### PR TITLE
PT methods fixed to be able to send pageSize on the request

### PR DESCRIPTION
**Pull request by hernantain**
**What**: Adding page_size parameter on `get_phishing_submissions_page` and `get_phishing_indicators_page` to be able to request pages with more than 25 elements. 

**Why**: Principally to replicate API parameters and behaviour but also needed on Demisto.

**How**: Adding a params dict on the requests.

### CODE REVIEW

File: trustar/phishing_triage_client.py [hunk 1]
```diff
@@ -193,7 +197,8 @@ def _get_phishing_indicators_page_generator(self, from_time=None, to_time=None,
         return CursorPage.get_cursor_based_page_generator(get_page, cursor=cursor)
 
     def get_phishing_indicators_page(self, from_time=None, to_time=None, normalized_indicator_score=None,
-                                     priority_event_score=None, enclave_ids=None, status=None, cursor=None):
+                                     priority_event_score=None, enclave_ids=None, status=None, cursor=None,
```
Comment (YaPeL) [inline trustar/phishing_triage_client.py:200]: We should just use a **kwargs

**Comment by YaPeL** [comment]
LGTM

**Comment by vanessacortes** [comment]
LGTM

File: trustar/phishing_triage_client.py (hunk 1 above)
Comment (michaelknopf) [reply trustar/phishing_triage_client.py:200]: Using **kwargs will impact the Sphinx docs.  Should look into how to improve this across the whole SDK, but until then it's safer to not change the pattern.

Comment (YaPeL) [reply trustar/phishing_triage_client.py:200]: Good to know, I approved the pr regardless of this

Comment (michaelknopf) [reply trustar/phishing_triage_client.py:200]: Thanks.  I agree the existing pattern is messy and creates a lot of boilerplate, though.

Comment (marcos-vivar) [reply trustar/phishing_triage_client.py:200]: > Using **kwargs will impact the Sphinx docs. Should look into how to improve this across the whole SDK, but until then it's safer to not change the pattern.

@YaPeL This pattern migration could be mapped as a technical debt ticket that we can tackle later on.

**Pull request merged**